RadixWallet/Core/FeaturePrelude/AddressView/AddressView.swift

```diff
@@ -89,7 +89,14 @@ extension AddressView {
 	}
 
 	private var image: Image {
-		Image(.copy)
+		let imageResource: ImageResource =
+			switch identifiable {
+			case .address:
+				.copy
+			case .transaction:
+				.iconLinkOut
+			}
```

Comment:
I would make this the computed property and put the `Image(...)` at the use site